globals.go

```diff
@@ -1148,6 +1156,9 @@ const (
 	Windows2016 WindowsVersion = "2016"
 )
 
+// ValidWindowsVersions contains all recognized container Windows versions.
+var ValidWindowsVersions = []WindowsVersion{Windows2016, Windows2019, Windows2022}
```

Comment:
This is only used in the unit tests for now to test all valid enum values, but they're useful sometimes to conveniently get a list of all valid enum values since Go doesn't have a built-in way to do this.

I didn't change the validation to a slice containment check because `utility.StringSliceContains` doesn't allow passing a `[]ContainerOS` (even though `ContainerOS` is a `string` enum) due to Go typing rules. If you still prefer the slice way though, I can modify the validation to iterate through the slice. I think it's not too important though, because it would be pretty uncommon for us to add/remove enum values OS/arch/Windows version.

Reply:
Ok SGTM, it's fine we can keep the validation switch statement. It's probably easier to read anyway.